includes/class-ur-plugin-updater.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Plugin Updater
+ *
+ * @author   WPEverest
+ * @category Admin
+ * @package  UserRegistration/Admin
+ * @version  1.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! class_exists( 'UR_AddOn_Updater', false ) ) {
+	include_once( dirname( __FILE__ ) . '/admin/updater/class-ur-addon-updater.php' );
+}
+
+/**
+ * UR_Plugin_Updater Class.
+ */
+class UR_Plugin_Updater {
+
+	/**
+	 * Plugin ID.
+	 * @var int
+	 */
+	private $plugin_id = 24; // Don't know how to fetch personal or developer plan over here. This ID is of personal plan :)
```

Comment:
Need to get the exact plugin plan like personal or developer. Don't know how to fetch personal or developer plan over here. CC @rabinstha any idea?

Reply:
Since we can bypass this without adding `item_id` param and using `EDD_BYPASS_NAME_CHECK` I am closing this :)